rasa/core/actions/forms.py

```diff
@@ -165,16 +166,46 @@ def _entity_mapping_is_unique(
         mapping_as_string = json.dumps(slot_mapping, sort_keys=True)
         return mapping_as_string in self._unique_entity_mappings
 
-    @staticmethod
+    def get_global_not_intents(self, domain: Domain) -> List[Text]:
+        """Returns a list of ignored intents as retrieved from `global_not_intent`.
+
+        Args:
+            domain: The current model domain.
+
+        Returns:
+            The value/s found in `global_not_intent` parameter in the `domain.yml`
+            (under forms).
+        """
+        if domain.__dict__["forms"]:
```

Comment:
why are we accessing the forms attribute via `__dict__`? Can't we just use `domain.forms`? Even cleaner would be something like `domain.get_global_not_intent(self.name()` to avoid breaking the [Law of Demeter](https://en.wikipedia.org/wiki/Law_of_Demeter)

Reply:
It's fine 👍🏻